docs/libraries/azure-resource-manager/rules/no-response-body.md

```diff
@@ -0,0 +1,98 @@
+---
+title: no-empty-model
+---
+
+```text title=- Full name-
+@azure-tools/typespec-azure-resource-manager/no-response-body
+
+```
+
+ARM response operations with status code 202 or 204 should not contain response body. Operations that are different for 202 and 204 should contain a response body.
+
+### For 202 and 204 status codes (response body should be empty)
+
+#### ❌ Incorrect
+
+```tsp
+model Test204BodyResponse {
```

Comment:
Since this is an ARM rule, I would change this to use our own constructs, rather than custom models with status code, for example:
```ArmNoContentResponse & {@body body: string}```